### DISCUSSION TITLE
Valitdate useFieldArray

### DISCUSSION THREAD

**Discussion by zhihli** [react-hook-form/react-hook-form#7601] [Q&A]
I want to validate if a input item contains duplicate values , then show a error message on input item,   here is my solulation [CSB](https://codesandbox.io/s/validate-fieldarray-24uzn),  Is there any other solulation?

**Comment by bluebill1049** [comment]
```tsx
useEffect(() => {
  // do your validation at group level and setError()
}, [fields])
```

**Reply by zhihli** [reply]
so quickly, thank your reply,  but it doesn't seem to work,  i want to trigger validation when input onChange

**Reply by bluebill1049** [reply]
then you can use `validate` function with `register`
